packages/themes/ecl/src/components/ecl-ec/input-radio.scss

```diff
@@ -21,15 +22,24 @@
 	}
 
 	fieldset .hint {
-		order: 2;
+		order: 4;
+		font-size: 0.875rem;
+		font-style: italic;
 	}
 
 	fieldset .kol-alert-wc {
 		order: 3;
 	}
 
 	fieldset .kol-input {
-		order: 4;
+		display: grid;
```

Comment:
Siehe Kommentar in Basis-Styles.

Bitte auch allgemein keine Styles in die Themes duplizieren, die bereits im Basis-Style definiert sind. Bitte auch andere Properties und Themes einmal auf Redundanzen prüfen.